cog/comment.py

```
@@ -48,7 +48,7 @@ def get_channels():  # 要特殊用途頻道的列表，這裡會用來判斷是
     # os.chdir("./")
     try:
         with open(
-            f"{os.getcwd()}/DataBase/server.config.json", "r", encoding="utf-8"
+            f"{os.getcwd()}/database/server.config.json", "r", encoding="utf-8"
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)